crates/txtx-cli/src/cli/mod.rs

```diff
@@ -60,32 +60,32 @@ struct Opts {
 
 #[derive(Subcommand, PartialEq, Clone, Debug)]
 enum Command {
-    /// List runbooks present in the current direcoty
+    /// List runbooks in the current directory
     #[clap(name = "ls", bin_name = "ls")]
     List(ListRunbooks),
-    /// Create a new Runbook
+    /// Create a new runbook
     #[clap(name = "new", bin_name = "new")]
     New(CreateRunbook),
-    /// Check the executability of a runbook
+    /// Check if a runbook can be executed
     #[clap(name = "check", bin_name = "check")]
     Check(CheckRunbook),
-    /// Run, runbook, run!
+    /// Execute a runbook
     #[clap(name = "run", bin_name = "run")]
     Run(ExecuteRunbook),
-    /// Display Documentation
+    /// Display documentation
     #[clap(name = "docs", bin_name = "docs")]
     Docs(GetDocumentation),
-    /// Start Txtx Language Server
+    /// Start the txtx language server
     #[clap(name = "lsp", bin_name = "lsp")]
     Lsp,
-    /// Start Txtx Server
+    /// Start the txtx server
     #[clap(name = "serve", bin_name = "serve")]
     #[cfg(feature = "txtx_serve")]
     Serve(StartServer),
     /// Snapshot management (work in progress)
     #[clap(subcommand)]
     Snapshots(SnapshotCommand),
-    /// Txtx cloud commands
+    /// txtx cloud commands
```

Comment:
```suggestion
    /// Txtx cloud commands
```
I think since it's the first, let's capitalize